sg/internal/runner/runner.go

```diff
@@ -15,6 +17,13 @@ var (
 
 // RunOnce uses key to ensure that fn runs exactly once and always returns the error from the initial run.
 func RunOnce(ctx context.Context, key string, fn func(context.Context) error) error {
+	before := time.Now()
+	defer func() {
+		if os.Getenv("SAGE_TRACE") == "true" {
+			elapsed := time.Since(before)
+			println(key, "took", elapsed.String())
```

Comment:
any reason for not using the logger?

Reply:
nope, I will change if this is something we actually want merged to master :)